blockstore.go

```diff
@@ -87,16 +87,20 @@ func newBlockStore(orchestrator, loggingEndpoint string) (blockstore.Blockstore,
 		},
 	}
 
+	saturnServiceClient := &http.Client{
+		Transport: &withUserAgent{RoundTripper: http.DefaultTransport},
+	}
+
 	return caboose.NewCaboose(&caboose.Config{
 		OrchestratorEndpoint: *oe,
-		OrchestratorClient:   http.DefaultClient,
+		OrchestratorClient:   saturnServiceClient,
 
 		LoggingEndpoint: *le,
-		LoggingClient:   http.DefaultClient,
+		LoggingClient:   saturnServiceClient,
```

Comment:
:information_source: @guanzo I think we've found the offender – we were using custom user-agent client only for saturn client, orchestrator, and logs used  default one. This PR (when merged) should fix this.